src/Engines/nativeEngine.ts

```diff
@@ -1771,6 +1773,38 @@ export class NativeEngine extends Engine {
         }
     }
 
+    /**
+     * Engine abstraction for createImageBitmap with Native implementation
+     * @param args parameters for createImageBitmap implementation
+     * @returns ImageBitmap
+     */
+    public createImageBitmap(...args: any[]): Promise<ImageBitmap>
+    {
+        return new Promise((resolve, reject) => {
+            if (args.length && Array.isArray(args[0]) && args[0].length) {
+                const image = this._native.createImageBitmap(args[0][0]);
+                if (image) {
+                    resolve(image);
+                } else if (reject) {
+                    reject();
+                }
+                return;
+            }
+            throw new Error(`Unsupported data for createImageBitmap.`);

```

Comment:
this is a bit odd, although this will translate to a reject call... why not just call reject here?